geode-assembly/src/integrationTest/resources/assembly_content.txt

```diff
@@ -0,0 +1,1245 @@
+LICENSE
+NOTICE
+README.md
+bin/gfsh
+bin/gfsh-completion.bash
+bin/gfsh.bat
+config/cache.xml
+config/gemfire.properties
+config/log4j2.xml
+javadoc/allclasses-frame.html
```

Comment:
Is it worthwhile to assert against individual javadoc files?

Reply:
I think so. Yes it will be an explicit process to update this file if javadocs are added or removed but considering they should only be for public APIs this should be infrequent. More importantly it will catch a mass delete or add of javadocs we didn't intend.

Reply:
This change makes sure that the on-disk layout of our release looks like we expect it to.